docs/spec/data-format.en-US.md

```diff
@@ -70,11 +70,11 @@ We suggest the following formats:
 
 To the users, the accuracy of time is not so important as the immediacy of the information. In the console platform, relative time is generally used for message and notification. And users tend to pay more attention to the unit of time, instead of working out the specific time point of publication.
 
-| Time                 | Display form                              |
-| -------------------- | ----------------------------------------- |
-| Less than 1 minute   | just now                                  |
-| Less than 1 hour     | N minutes ago                             |
-| Within 24 hours      | N hours ago                               |
+| Time                 | Display form                                |
+| -------------------- | ------------------------------------------- |
+| Less than 1 minute   | just now                                    |
+| Less than 1 hour     | N minutes ago                               |
+| Within 24 hours      | N hours ago                                 |
```

Comment:
Not neccessary to change these.

Reply:
@afc163 Why? We can't set the background colour for filling in invalid fields now

Reply:
I mean these markdown files shouldn't be changed.

<img width="676" alt="image" src="https://user-images.githubusercontent.com/507615/96143494-b63bfc80-0f35-11eb-8c2e-5b2ebe0cf445.png">

Reply:
These files were pulled from the `master` branch when resolving conflict in my PR. I did not change them directly
https://github.com/ant-design/ant-design/commit/8b3f277abae50a6e517ff5a96e3ca82f838f7683

Reply:
It should not exist in this PR, any way.

Reply:
@afc163 ok. problem resolved